Compilation Error on Raspberry Pi 3 - 64 Bit

Compilation (make) fails with the following error:
```
[  9%] Building CXX object src/CMakeFiles/phylanx_component.dir/execution_tree/primitives/slice.cpp.o
[  9%] Building CXX object src/CMakeFiles/phylanx_component.dir/execution_tree/primitives/slice_node_data.cpp.o
In file included from /opt/repos/phylanx/src/execution_tree/primitives/slice_node_data.cpp:10:
In file included from /opt/repos/phylanx/phylanx/execution_tree/primitives/slice_node_data_0d.hpp:11:
In file included from /opt/repos/phylanx/phylanx/execution_tree/primitives/detail/slice_assign.hpp:10:
/opt/repos/phylanx/phylanx/execution_tree/primitives/detail/slice_identity.hpp:88:24: error: no template named 'DynamicTensor' in
      namespace 'blaze'; did you mean 'DynamicVector'?
                blaze::DynamicTensor<T>{std::forward<View>(view)}};
                ~~~~~~~^~~~~~~~~~~~~
                       DynamicVector
/usr/include/blaze/math/dense/DynamicVector.h:197:7: note: 'DynamicVector' declared here
class DynamicVector
      ^
1 error generated.
make[2]: *** [src/CMakeFiles/phylanx_component.dir/build.make:609: src/CMakeFiles/phylanx_component.dir/execution_tree/primitives/slice_node_data.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1134: src/CMakeFiles/phylanx_component.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
```
uname -a
Linux localhost.localdomain 5.0.3-200.fc29.aarch64 #1 SMP Tue Mar 19 14:56:09 UTC 2019 aarch64 aarch64 aarch64 GNU/Linux

clang++ --version
clang version 7.0.1 (Fedora 7.0.1-6.fc29)

commit e5ab9dc95fd2c2aa4c6b131d80e4edd92f7eab50 

cmake -DHPX_DIR=/opt/hpx -DCMAKE_INSTALL_PREFIX=/opt/phylanx ..

Name         : blaze-devel
Version      : 3.4
Release      : 1.fc29

--- a/phylanx/execution_tree/primitives/detail/slice_assign.hpp
+++ b/phylanx/execution_tree/primitives/detail/slice_assign.hpp
@@ -284,6 +284,7 @@ namespace phylanx { namespace execution_tree { namespace detail
             return ir::node_data<T>{std::move(data)};
         }
 
+#if defined(PHYLANX_HAVE_BLAZE_TENSOR)
         template <typename Data, typename View, typename F = always_true>
         ir::node_data<T> tensor(Data& data, View&& view, F const& f = F{}) const
         {
@@ -308,6 +309,7 @@ namespace phylanx { namespace execution_tree { namespace detail
             }
             return ir::node_data<T>{std::move(data)};
         }
+#endif
     };
 }}}
 

--- a/phylanx/execution_tree/primitives/detail/slice_identity.hpp
+++ b/phylanx/execution_tree/primitives/detail/slice_identity.hpp
@@ -80,6 +80,7 @@ namespace phylanx { namespace execution_tree { namespace detail
                 blaze::DynamicMatrix<T>{std::forward<View>(view)}};
         }
 
+#if defined(PHYLANX_HAVE_BLAZE_TENSOR)
         template <typename Data, typename View, typename F = always_true>
         ir::node_data<T> tensor(
             Data const&, View&& view, F const& f = F{}) const
@@ -87,6 +88,7 @@ namespace phylanx { namespace execution_tree { namespace detail
             return ir::node_data<T>{
                 blaze::DynamicTensor<T>{std::forward<View>(view)}};
         }
+#endif
     };
 }}}
 

